.github/CODEOWNERS

```diff
@@ -335,7 +335,7 @@
 /comp/core/workloadmeta @DataDog/container-platform
 /comp/forwarder/eventplatform @DataDog/agent-log-pipelines
 /comp/forwarder/eventplatformreceiver @DataDog/agent-log-pipelines
-/comp/forwarder/orchestrator @DataDog/agent-log-pipelines
+/comp/forwarder/orchestrator @DataDog/container-platform
```

Comment:
This should be owned by @DataDog/container-app

Reply:
Fixed!